File: src/main/java/com/ptah/dto/userprofiling/UserDto.java
package com.ptah.dto.userprofiling;

import com.ptah.common.BaseDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UserDto extends BaseDto<UserDto> {
    private String login;
    private String nickname;


}

File: backend/src/main/kotlin/com/github/jarryzhou/ptah/dto/userprofiling/UserDto.kt
package com.github.jarryzhou.ptah.dto.userprofiling

import com.github.jarryzhou.ptah.common.BaseDto

class UserDto(
    var login: String? = null,
    var nickname: String? = null,
    var authorities: List<String>? = emptyList()
) : BaseDto()
